aten/src/ATen/native/native_functions.yaml

```diff
@@ -9905,3 +9905,24 @@
 - func: unflatten_dense_tensors(Tensor flat, Tensor[] tensors) -> Tensor[]
   variants: function
   python_module: nn
+
+# Functions which do no work and simply return `self`. Used to measure overhead.
```

Comment:
If a function return self, then the output alias the input and it should have `Tensor(a)` between them. These signatures are not valid right now.

Reply:
This was deliberate, as I wanted to simulate the behavior of a kernel which `could` return a new Tensor (from the point of view of the system); it just happens to do nothing. It might be moot if the perf is basically the same. I'll test that today.

Reply:
Turns out it makes no difference, so I've updated the schema. Thanks!